packages/generate_package.sh

```diff
@@ -95,7 +95,9 @@ build_pkg() {
         -e IS_STAGE="${IS_STAGE}" \
         -e WAZUH_BRANCH="${BRANCH}" \
         ${CUSTOM_CODE_VOL} \
+        -v ${DOCKERFILE_PATH}:/home:Z \
         ${CONTAINER_NAME}:${DOCKER_TAG} \
+        /home/build.sh \
```

Comment:
Can we modify this line and turn it into a variable which we set earlier in the code?
Such as `${ENTRYPOINT}`
```suggestion
       ${ENTRYPOINT} \
```

Reply:
Done